k8s/charts/openebs/values.yaml

```diff
@@ -81,6 +81,7 @@ ndm:
     count: "1"
   filters:
     excludeVendors: "CLOUDBYT,OpenEBS"
+    includePaths: ""
```

Comment:
This option needs to be documented in the README.

Reply:
@kmova  The readme for the charts right? Will do it .

Reply:
Added in README.